pyvista/core/utilities/geometric_objects.py

```diff
@@ -507,7 +507,11 @@ def MultipleLines(points=[[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]]):
     >>> mesh = pyvista.MultipleLines(
     ...     points=[[0, 0, 0], [1, 1, 1], [0, 0, 1]]
     ... )
-    >>> mesh.plot(color='k', line_width=10)
+    >>> plotter = pyvista.Plotter()
+    >>> actor = plotter.add_mesh(mesh)
```

Comment:
```suggestion
    >>> actor = plotter.add_mesh(mesh, color='k', line_width=10)
```
I added options to match the appearance of the mesh with that before the change. Can you confirm that this is not a problem and approve the change?